packages/chatbot-server-mongodb-public/src/mongoDbMetadata.ts

```diff
@@ -0,0 +1,192 @@
+import { id } from "common-tags";
+import { z } from "zod";
+
+export const MongoDbProductSchema = z.object({
+  id: z.string().describe("Unique identifier for the product"),
+  name: z.string().describe("Human-friendly name of the product"),
+  description: z
+    .string()
+    .optional()
+    .describe("Brief description of the product"),
+  versions: z
+    .array(z.string())
+    .optional()
+    .describe("List of versions of the product"),
+  parentProductId: z.string().optional().describe("`id` of the parent product"),
+});
+export type MongoDbProduct = z.infer<typeof MongoDbProductSchema>;
+
+export const mongoDbProducts = [
```

Comment:
How did we come up with this list? Are we sure there aren't any significant gaps?

Reply:
i am not sure there aren't significant gaps. the list comes from the products mentioned in the old extractMongoDbMetadata function. let me make a ticket to investigate/refine this list further

Reply:
https://jira.mongodb.org/browse/EAI-505